charts/sonarqube/templates/apply-aml-github-job.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.aml }}
+{{- if .Values.aml.github }}
+{{- if and .Values.aml.github.appId .Values.aml.github.url .Values.aml.github.clientId .Values.aml.github.clientSecret .Values.aml.github.key .Values.aml.github.privateKey }}
```

Comment:
i understand why you want to catch everything but following the principals from the rest of this helm chart a simple `enable` flag should be enough. 
A failing container is more easy to debug than a job that is never created in the first place if a user in the future does not provide all required values